content/theme/11straps.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "11straps"
+github: https://github.com/holger1411/11straps
+demo: https://11straps.com/
```

Comment:
Demo link must go directly to demo, not sales/company page.

Reply:
@JugglerX Changed the URL to the demos URL